account_invoice_mode_at_shipping/tests/test_invoice_mode_at_shipping.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo.tests.common import SavepointCase
+
+
+class TestInvoiceModeAtShipping(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
```

Comment:
```suggestion
        super().setUpClass()
        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```
you could move this and the creation of sale order in the base class (which is not used here).
Otherwise, remove the base class.

Reply:
I removed the base class